sync2/rangesync/rangesync.go

```diff
@@ -702,8 +671,8 @@ RECV_LOOP:
 // Run performs sync reconciliation run using specified Conduit to send and receive
 // messages.
 func (rsr *RangeSetReconciler) Run(c Conduit) error {
-	rsr.log.Debug("begin set reconciliation")
-	defer rsr.log.Debug("end set reconciliation")
+	rsr.logger.Debug("begin set reconciliation")
```

Comment:
here too.. not much context

Reply:
This is a useful "delimiter" when we enable verbose debug logging.